Artifacts/windows-install-powershell-module/Artifactfile.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Rename artifact folder to `windows-powershellmodule` to follow the convention used for other PS artifacts.